tests/tools/interpreter_test.py

```diff
@@ -0,0 +1,16 @@
+from ...tools.interpreter import Interpreter
+import logging
+import unittest
+from ...tests.kernel.wave.wave_gemm_test import Test as wave_gemm_test
```

Comment:
Yeah, unfortunately, you cannot do relative imports if you are not inside the package, just copypaste gemm here?

Reply:
okay will do.